javascript-data-model/order-history.js

```diff
@@ -0,0 +1,97 @@
+var orderHistory = [
+  {
+    orderNumber: '114-3941689-8772232',
+    orderDate: '2020-08-04',
+    orderDetailsURL: '/order/114-3941689-8772232',
+    invoiceURL: '/invoice/order/8772232',
+    orderPrice: 34.00,
+    deliveryDate: '2020-08-08',
+    returnDate: '2020-11-07',
+    shipRecipient: 'JS Master',
+    itemList: [
+      {
+        itemPrice: 31.55,
+        itemURL: '/JavaScript-for-impatient-programmers/Bdfdkq',
+        book: {
+          title: 'JavaScript for impatient programmers',
+          author: 'Axel Rauschmayer',
+          prefix: 'Dr.',
+          imageURL: './images/898290389278'
+        }
+      }
+    ]
+  },
+
+  {
+    orderNumber: '113-9984268-1270257',
+    orderDate: '2020-07-19',
+    orderDetailsURL: '/order/113-9984268-1270257',
+    invoiceURL: '/invoice/order/1270257',
+    orderPrice: 44.53,
+    deliveryDate: '2020-07-20',
+    returnDate: '2020-08-19',
+    shipRecipient: 'JS Master',
+    itemList: [
+      {
+        itemPrice: 41.33,
+        itemURL: '/The-Timeless-Way-of-Building/Ttsdqqw',
+        book: {
+          title: 'The Timeless Way of Building',
+          author: 'Christopher Alexander',
+          imageURL: './images/242394023489'
+        }
+      }
+    ]
+  },
+
+  {
+    orderNumber: '114-2875557-9059409',
+    orderDate: '2020-07-04',
+    orderDetailsURL: '/order/114-2875557-9059409',
+    invoiceURL: '/invoice/order/9059409',
+    orderPrice: 17.22,
+    deliveryDate: '2020-07-07',
+    returnDate: '2020-08-05',
+    shipRecipient: 'JS Master',
+    itemList: [
+      {
+        itemPrice: 15.98,
+        itemURL: '/GameCube-Controller-Adapter-Cloud-Store/Gcw2ss',
+        gamePeripheral: {
+          name: 'GameCube Controller Adapter. Super Smash Bros Switch GameCube Adapter for Wii U, PC. Support Turbo and Vibration Features. No Driver and No-Lag Gamecube Adapter',
+          imageURL: './images/3458353899898'
+        }
+      }
+    ]
+  },
+
+  {
+    orderNumber: '113-2883177-2648248',
+    orderDate: '2020-07-03',
+    orderDetailsURL: '/order/113-2883177-2648248',
+    invoiceURL: '/invoice/order/2648248',
+    orderPrice: 138.93,
+    deliveryDate: '2020-07-05',
+    returnDate: '2020-08-04',
+    shipRecipient: 'JS Master',
+    itemList: [
+      {
+        itemPrice: 94.95,
+        itemURL: '/GameCube-Controller-Smash/TLLDFD',
+        gamePeripheral: {
+          name: 'GameCube Controller - Super Smash Bros. Edition (Nintendo Switch)',
+          imageURL: './images/90880942323432'
+        }
```

Comment:
How would somebody who goes to use this data know to look for a `gamePeripheral` or `book` property? Instead, I would give the orders a `type` or `category` property with a value of what that item is, so that the format you store the data in remains consistent.